src/Xamarin.Android.Build.Tasks/Tasks/AsyncTask.cs

```diff
@@ -114,6 +115,14 @@ protected void LogError (string message, params object[] messageArgs)
 			LogError (string.Format (message, messageArgs));
 		}
 
+		public void LogErrorFromException (Exception exception)
+		{
+			var sb = new StringBuilder ();
+			sb.Append (exception.Message);
+			sb.Append (exception.StackTrace);
+			LogError (sb.ToString ());
```

Comment:
what about exception.InnerException ? Would it be easier to call `LogError (exception.ToString ())` to ensure all data is captured?